modules/async-icq/testing/demo-simapp/x/interquery/types/types.go

```diff
@@ -0,0 +1 @@
+package types
```

Comment:
i don't think this file is necessary anymore

Reply:
resolved in 3babd4a4c754d3c1da0ae43806597e2cbf270a93